src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,152 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+
+    private static class Node<T> {
+        private T item;
+        private Node<T> next;
+        private Node<T> prev;
+
+        public Node(T item) {
```

Comment:
"When creating a new Node, pass all the fields to Node constructor" - from common mistakes file

Reply:
I would leave like that in order not to assign null to prev or next or both every time I create an object

Reply:
[When creating a new Node, pass all the fields to Node constructor](https://mate-academy.github.io/jv-program-common-mistakes/java-core/collections/linked-list.html#when-creating-a-new-node-pass-all-the-fields-to-node-constructor)